packages/app/src/cli/constants.ts

```diff
@@ -266,6 +268,7 @@ export const extensionGraphqlId = (type: ExtensionTypes) => {
     case 'order_discounts':
     case 'shipping_discounts':
     case 'payment_methods':
+    case 'payment_customization':
     case 'shipping_rate_presenter':
       // As we add new extensions, this bug will force us to add a new case here.
```

Comment:
Super confusing comment 🤔  What bug?

Reply:
This is an old comment, no longer applies